app/views/admin/conversations/index.html.erb

```diff
@@ -19,8 +19,8 @@
       <% table.with_header t(".business") %>
       <% table.with_header t(".developer") %>
       <% table.with_header t(".started") %>
-      <% table.with_header "#{t(".replied")}?" %>
-      <% table.with_header "#{t(".email")}?" %>
+      <% table.with_header "#{t(".replied")}?", "#{t(".replied_tooltip")}" %>
+      <% table.with_header "#{t(".email")}?", "#{t(".email_tooltip")}" %>
```

Comment:
It looks like CI is failing because you don't need to interpolate if you are using the entire string.

You can change this to match a few lines above: `t(".replied_tooltip")`.

Reply:
Thanks! You are totally right.
Last night, I was more focused on learning View Components and didn't catch that the interpolation was required only for the '?'.

Reply:
All good! Thanks for the quick fix.